### DISCUSSION TITLE
Manual override not working

### DISCUSSION THREAD

**Discussion by NashJames** [github-linguist/linguist#5887] [Classification] [answered]
I’ve tried overriding the SVG detection for this [repo](https://github.com/NashJames/Potions) to create a language breakdown. 2 variations I tried:

`*.svg linguist-detectable`
`src/*.svg linguist-detectable`

Not sure if this is a bug or my .gitattributes syntax is wrong?

**Comment by lildude** [comment, marked as answer]
> Not sure if this is a bug or my .gitattributes syntax is wrong?

Neither. Modifying `.gitattributes` alone is not enough to trigger a re-analysis. You'll need to push another change to another file for the analysis to take place and take into account your changes.

**Reply by NashJames** [reply]
So I guess mine is a slightly more interesting case. I'm guessing it only actually triggers on a programming or markup file? Nevermind what changes you've made to the `.gitattributes`

I've neither of these so I made a dummy html file, let that trigger the job. It worked, but said 100% HTML (not including the svg override).

I deleted the file to make sure, and as expected, the linguist breakdown disappeared.

Any ideas?

**Reply by lildude** [reply]
> Any ideas?

Yup. More t's; less r's 😉:

![CleanShot 2022-05-10 at 09 02 30@2x](https://user-images.githubusercontent.com/627280/167579808-c1939feb-9597-429e-9769-bd9d08e632ea.png)

**Reply by NashJames** [reply]
hahaha that's hilarious. thanks
